src/ext.js

```diff
@@ -58,12 +58,12 @@ export default function ext(/* env */) {
               },
               show: data => checkShow(data, 'field'),
             },
-            softLock: {
-              type: 'boolean',
-              ref: 'softLock',
-              label: 'Overwrite locked selections',
-              defaultValue: false,
-              show: data => checkShow(data, 'softLock'),
+            variable: {
+              type: 'string',
+              ref: 'variable',
+              component: 'string',
```

Comment:
I think we can keep this as a string component, since we don't need expressions for the name.
Or maybe it should be a dropdown, since you can only change variables not create new ones.

Reply:
Not sure if you would need an expression for the name, maybe we should check that. Otherwise for now it's okay I think. If we go for a dropdown then we definitely don't have the option for expression anymore. And if we go that road then we should probably have a searchable dropdown anyway

Reply:
just checked and the old button has an expression component